htmd/parameterization/test.py

```diff
@@ -7,12 +7,77 @@
 import sys
 import shutil
 import unittest
-import numpy as np
 from subprocess import call
 
 from htmd.home import home
 from htmd.util import tempname
 
+
```

Comment:
This file cannot be changed, except the atom names.

Reply:
Sorry? I don't understand this comment

Reply:
Do not change `htmd/parameterization/test.py` in this PR, except for the atom names! It seems your are pick up the old code.